cpp/src/join/conditional_join.cu

```diff
@@ -377,16 +377,12 @@ conditional_inner_join(table_view const& left,
                        table_view const& right,
                        ast::expression const& binary_predicate,
                        std::optional<std::size_t> output_size,
+                       rmm::cuda_stream_view stream,
                        rmm::device_async_resource_ref mr)
 {
   CUDF_FUNC_RANGE();
-  return detail::conditional_join(left,
-                                  right,
-                                  binary_predicate,
-                                  detail::join_kind::INNER_JOIN,
-                                  output_size,
-                                  cudf::get_default_stream(),
-                                  mr);
+  return detail::conditional_join(
+    left, right, binary_predicate, detail::join_kind::INNER_JOIN, output_size, stream, mr);
```

Comment:
Since the call to `cudf::get_default_stream()` is removed, the corresponding header ` <cudf/utilities/default_stream.hpp>` is no longer needed in this source file and can be removed too.

Reply:
thanks, done!